### DISCUSSION TITLE
Thanks Rocky

### DISCUSSION THREAD

**Discussion by kcabral817** [MarimerLLC/csla#1666] [General]
I just found the Thanks discussion category and wanted to thank you for your kindness when helping folks.  The one thing I don't like about Stack Overflow is the attitude and condescension some folks have about why something is done that _way_, etc.  Here, you seem not to lose patience with people and are quite helpful.  Take care.  And again, thank you for your support.

**Comment by Chicagoan2016** [comment]
@kcabral817 , thank you for bringing this up! I agree about the Stackoverflow. I also agree what you said about @rockfordlhotka, I consider myself his student and I wish we have more people like @rockfordlhotka  in this world. This planet will become a much better place.

I do feel like we need to have another platform in addition to Stackoverflow, monopoly is not good, even when it comes to Q&A platform about software development.

Kind Regards

**Comment by rockfordlhotka** [comment]
You are welcome, I really enjoy working with the CSLA community - so many great people!
